common/src/main/java/tech/pegasys/web3signer/common/JacksonSerializers.java

```diff
@@ -128,4 +142,18 @@ public void serialize(
       gen.writeString(value.toString());
     }
   }
+
+  /**
+   * Safely get value as string from parser, handling potential exceptions.
+   *
+   * @param p JsonParser
+   * @return String representation of the value or placeholder if extraction fails
+   */
+  private static String safeGetValueAsString(final JsonParser p) {
+    try {
+      return p.getValueAsString();
+    } catch (Exception e) {
+      return "<unable to extract value>";
```

Comment:
This will eventually result in `Failed to parse <unable to extract value> as a UInt64` if p.getValueAsString fails.